libraries/testing/contracts/sysio.roa/sysio.roa.abi

```diff
@@ -270,40 +396,67 @@
         {
             "name": "activateroa",
             "type": "activateroa",
-            "ricardian_contract": "---\nspec_version: \"0.2.0\"\ntitle: Activate ROA Contract\nsummary: 'Activates ROA setting {{nowrap max_ram_bytes}} and ram price {{nowrap ram_byte_price}}'\nicon: https://raw.githubusercontent.com/eosnetworkfoundation/eos-system-contracts/main/contracts/icons/admin.png#9bf1cec664863bd6aaac0f814b235f8799fb02c850e9aa5da34e8a004bd6518e\n---\n\n{{$action.account}} activates the ROA contract and sets sysio.roa's {{max_ram_bytes}} to the networks capacity of RAM. It also sets the price of 1 byte of RAM {{ram_byte_price}}. Finally it adds system accounts to the reslimit table."
+            "ricardian_contract": ""
```

Comment:
did we intend to drop the ricardian contract payloads from the ABI?

Reply:
Doesn't really matter for the tests. We build the tests via cmake in wire-sysio which doesn't gen the ricardian contracts.